app/javascript/react/components/AnimalIndex.js

```diff
@@ -31,7 +33,15 @@ const AnimalIndex = (props) => {
 
   return (
     <div className="grid-container">
-      <h1>Animals</h1>
+      <div className="grid-x grid-margin-x">
+        <div className="cell small-6 medium-4">
+          <img src={IndexBlathers} alt="blathers" className="index-blathers"/>
```

Comment:
Make sure you push info like this up to Heroku more often. Sometimes assets get weird and not show up on Heroku. - this is a note not a correction!